lib/screens/home.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/helpers/spacer_height.dart';
+import 'package:flutter_internship/screens/component.dart';
+import 'package:flutter_internship/screens/layout.dart';
+import 'package:flutter_internship/screens/mobile_app.dart';
+import 'package:flutter_internship/widgets/expanded_button.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              ExpandedButton(
+                backgroundColor: Colors.purple,
+                foregroundColor: Colors.white,
+                buttonText: 'Layout',
+                onTap: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                      builder: (context) => const LayoutScreen(),
+                    ),
+                  );
+                },
+              ),
+              const SpacerHeight(),
+              ExpandedButton(
+                backgroundColor: Colors.blue,
+                foregroundColor: Colors.white,
+                buttonText: 'Components',
+                onTap: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                      builder: (context) => const ComponentClass(),
+                    ),
+                  );
+                },
+              ),
+              const SpacerHeight(),
+              ExpandedButton(
+                backgroundColor: Colors.orange,
+                foregroundColor: Colors.white,
+                buttonText: 'Mobile App',
+                onTap: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                      builder: (context) => const MobileAppClass(),
+                    ),
+                  );
+                },
+              ),
```

Comment:
Space between buttons is still bigger then in Figma and they should't go edge to edge with screen.